pkg/synthetictests/operators.go

```diff
@@ -39,7 +39,7 @@ func testOperatorStateTransitions(events monitorapi.Intervals, conditionTypes []
 	}
 	duration := stop.Sub(start).Seconds()
 
-	knownOperators := allOperators(events)
+	knownOperators := sets.NewString(platformidentification.KnownOperators.List()...)
```

Comment:
allOperators was already using this list, but then adding any new operators seen in events.

This change will have the effect of killing off the control-plane-machine-set test entirely, because it's not in https://github.com/openshift/origin/blob/master/pkg/synthetictests/platformidentification/operator_mapping.go#L87-L120

Can we update that file too? I also think you can get rid of the intermediate variable and just reference `platformidentification.KnownOperators.List()` on line 46 here